src/test/java/org/numenta/nupic/network/NetworkTestHarness.java

```diff
@@ -26,6 +26,8 @@
 
 import org.numenta.nupic.Parameters;
 import org.numenta.nupic.Parameters.KEY;
+import org.numenta.nupic.algorithms.CLAClassifier;
```

Comment:
In general I try to not have any "yellow" markers (in Eclipse) - basically no compiler warnings being generated.